org.eclipse.xtext.tests/testdata/wizard-expectations/mavenTychoJ11/mavenTychoJ11.parent/mavenTychoJ11/pom.xml

```diff
@@ -36,26 +36,6 @@
 					<cleanupDaemonThreads>false</cleanupDaemonThreads><!-- see https://bugs.eclipse.org/bugs/show_bug.cgi?id=475098#c3 -->
 				</configuration>
 				<dependencies>
-					<dependency>
-						<groupId>org.eclipse.emf</groupId>
-						<artifactId>org.eclipse.emf.mwe2.launch</artifactId>
-						<version>${mwe2Version}</version>
-					</dependency>
-					<dependency>
-						<groupId>org.eclipse.xtext</groupId>
-						<artifactId>org.eclipse.xtext.common.types</artifactId>
-						<version>${xtextVersion}</version>
-					</dependency>
-					<dependency>
-						<groupId>org.eclipse.xtext</groupId>
-						<artifactId>org.eclipse.xtext.xtext.generator</artifactId>
-						<version>${xtextVersion}</version>
-					</dependency>
-					<dependency>
-						<groupId>org.eclipse.xtext</groupId>
-						<artifactId>org.eclipse.xtext.xbase</artifactId>
-						<version>${xtextVersion}</version>
-					</dependency>
```

Comment:
in the log i still see

``` 
2021-07-21T19:14:20.5006666Z 82   WARN  StandaloneSetup    - Skipping conflicting project org.apache.commons.io at 'archive:file:/home/runner/.m2/repository/commons-io/commons-io/2.5/commons-io-2.5.jar!/' and using 'archive:file:/home/runner/.m2/repository/p2/osgi/bundle/org.apache.commons.io/2.8.0.v20210415-0900/org.apache.commons.io-2.8.0.v20210415-0900.jar!/' instead.
```
wonder where this is coming from

Reply:
maybe from the exec plugin itself?

Reply:
That's not the only occurrence, is it? I've seen these warnings for ages myself, I'm wondering if they are due now to

```xml
<includePluginDependencies>true</includePluginDependencies>
```

which is still there (see my previous comment)

Reply:
looks like this is not working in all cases

Reply:
i also dont understand if this also referrs to the "extra" plugin dependencies

Reply:
yes it does. so it wont find the antlr one and use the facade instead
https://pipelines.actions.githubusercontent.com/VGbhG7MqInqDfxY0bKv0qAuPeX3d22owq0UvtAsHMpYjQuu8bd/_apis/pipelines/1/runs/556/signedlogcontent/27?urlExpires=2021-07-23T05%3A09%3A55.7351874Z&urlSigningMethod=HMACV1&urlSignature=lnqMg85bgGGidvw73PGx1a8O5X4CiUjH12Uk2%2BjKvUE%3D
again something i have ZERO idea if it is any good idea

Reply:
executableDependency s dont seem to support versions

Reply:
can we postpone this issue?

Reply:
I think I lost you here... what is the problem?

Reply:
solve the antlr problem separately.

Reply:
> in the log i still see
> 
> ```
> 2021-07-21T19:14:20.5006666Z 82   WARN  StandaloneSetup    - Skipping conflicting project org.apache.commons.io at 'archive:file:/home/runner/.m2/repository/commons-io/commons-io/2.5/commons-io-2.5.jar!/' and using 'archive:file:/home/runner/.m2/repository/p2/osgi/bundle/org.apache.commons.io/2.8.0.v20210415-0900/org.apache.commons.io-2.8.0.v20210415-0900.jar!/' instead.
> ```
> 
> wonder where this is coming from

@cdietrich @szarnekow 
Just for the record, after additional experiments, these bad warnings go away by removing

```xml
<includePluginDependencies>true</includePluginDependencies>
```

and by relying only on additional.bundles

Reply:
yes. but that needs another source for the antlr dependency.
i dont trust our it department and want to avoid to it being ddosed by maven builds

Reply:
Yes, I remember that. I just wanted to confirm that the warnings are definitely due to mixing plugin dependencies (Maven deps) with Eclipse bundles. From what I recall, we were not completely sure about that